Editor/Elements/Graph/Edge.cs

```diff
@@ -11,8 +11,8 @@ public class Edge : BaseEdge {
         private const float k_InterceptWidth = 6.0f;
         private const float k_EdgeLengthFromPort = 12.0f;
         private const float k_EdgeTurnDiameter = 20.0f;
-        private const int k_DefaultEdgeWidth = 2;
-        private const int k_DefaultEdgeWidthSelected = 2;
+        private const float k_DefaultEdgeWidth = 2;
```

Comment:
i guess float gives us a more granular way to define line thickness? This is a breaking change for our NewGraph framework. But if you clarify the importance of this I can think of making the aprropiate changes.

Reply:
It's exactly that, float adds a larger range of possible line thickness. Which is pretty important when working with values that, imo, should probably be between 1 and 3. And 1 seems too small and 3 seems too large.

The constants above could stay int since they are local/private. I just assumed it was a mistake/oversight so I changed those as well. The important change is with `EdgeWidthUnselected` and `EdgeWithSelected`. For example, I do the following:

```csharp
public override float EdgeWidthUnselected { get; } = 1.5f;
public override float EdgeWidthSelected { get; } = 2.0f;
```